pages/techgenius/tg-router-config.md

```diff
@@ -35,7 +35,7 @@ _Via Command prompt_. Open the CMD window (press Windows button + R or search th
  
 _Via the Graphical User Interface_. Open Control Panel > Network and Sharing Center > Wi-Fi Connections (click link) > Details to get this dialogue box:
 
-![Windows GUI IP screen shot](images/tg-wifi-gui-config.png)
+![Windows GUI IP screen shot](images/wifi-gui-config.png)
```

Comment:
@Dashlorde think we want to change the file name by moving the file to the new name
`git mv wifi-gui-config.png tg-wifi-gui-config.png`